combirepo/binfmt.py

```diff
@@ -96,11 +96,30 @@ def register(architecture, qemu_executable_path):
     binfmt_register_path = os.path.join(binfmt_directory, "register")
     check.file_exists(binfmt_register_path)
 
-    binfmt_name = get_name(architecture)
+    binfmt_name = __get_name(architecture)
     binary_format = ":{0}:M::{1}:{2}:{3}:{4}".format(binfmt_name,
                                                      binfmt_magic[binfmt_name],
                                                      binfmt_mask[binfmt_name],
                                                      qemu_executable_path,
                                                      binfmt_flag[qemu_type])
-    register_command = ["echo", "\"" + binary_format + "\"", "|", "sudo", "tee", binfmt_register_path]
-    hidden_subprocess.call("Register binary formats.", register_command)
+    with open(binfmt_register_path, 'w') as binfmt_register:
+        binfmt_register.write(binary_format)
+
+
+def __parse_args():
+    """ customize arguments """
+    parser = argparse.ArgumentParser(description='Register qemu in binfmt_misc.')
+    parser.add_argument('--arch', '-a', default='armv7l', help='target architecture')
+    parser.add_argument('--qemu', '-q', default='/usr/bin/qemu', help='path to qemu executable')
```

Comment:
I think `qemu-arm` binary is better default if we have `armv7l` as default architecture.

Reply:
Done